/**:
  ros__parameters:
    node_params:
      use_raw: [false, false, false, false, false, false] # per camera: if true, use raw image data, otherwise use compressed images
    interface_params:
      map_colors: [0.3922, 0.5843, 0.9294,  # divider: cornflowerblue
                   1.0, 0.0, 0.0,           # ped_crossing: red  
                   0.4392, 0.5020, 0.5647]  # boundary: slategrey
    model_params:
      plugins_path: "$(var plugins_path)/plugins/libautoware_tensorrt_plugins.so"
      default_command: 2  # 0: TURN RIGHT, 1: TURN LEFT, 2: KEEP FORWARD
      map_confidence_thresholds: [0.9, 0.9, 0.9]
      object_confidence_thresholds: [0.5, 0.5, 0.9, 0.5, 0.5, 0.5, 0.5, 0.5, 0.9, 0.5]
    sync_params:
      front_camera_id: 0  # ID of the front camera (anchor camera) for synchronization
      sync_tolerance_ms: 100.0  # Synchronization tolerance in milliseconds
